.github/workflows/code-static-analysis.yml

```diff
@@ -35,14 +35,14 @@ jobs:
         uses: actions/checkout@v4
       - name: Set up Python 3.12
         if: ${{ matrix.language == 'python' }}
-        uses: actions/setup-python@v5.4.0
+        uses: actions/setup-python@v5
         with:
           python-version: '3.12'
```

Comment:
Should we update this too?

Reply:
Dependabot updates this action every couple of months.
I believe it's safe to expect that the `python-version` parameter will not be changed in minor versions.

So keeping a major will make it a yearly update instead.

Reply:
Let's try the latest ;) Updated.